app/src/main/java/com/github/studenttimetracker/MainActivity.java

```diff
@@ -61,14 +55,21 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                     switch (item.getItemId()) {
                         case R.id.nav_settings:
                             intent = new Intent(getApplicationContext(), SettingsActivity.class);
-                            startActivity(intent);
                             break;
                         case R.id.nav_info:
                             intent = new Intent(getApplicationContext(), InfoActivity.class);
-                            startActivity(intent);
+                            break;
+                        case R.id.nav_contact:
+                            intent = new Intent(Intent.ACTION_SENDTO);
+                            String uriText = "mailto:dominos55555@gmail.com" +
```

Comment:
Maila pewnie będzie trzeba wpisać do string.xml żeby było bardziej po Androidowemu, ale to raczej na  końcu😛

Reply:
i tak i nie, bo myślałem o tym - ale gdy wyciąga się wartość z strings.xml z tego to wychodzi wielki int (w stylu 1032421) zamiast stringa. Plik xml ładnie to odczytuje, ale javie nie pasowały wartości xd